packages/router/src/directives/router_outlet.ts

```diff
@@ -27,6 +27,19 @@ import {PRIMARY_OUTLET} from '../shared';
  * <router-outlet name='right'></router-outlet>
  * ```
  *
+ * Named outlets can be the targets of secondary routes.
+ * The route configuration for a secondary route has a third property, `outlet:
+ * <target_outlet_name>`. Secondary routes are independent of each other, but work in combination
+ * with other routes. Using named outlets and secondary routes, you can target multiple outlets with
+ * multiple routes in the same `RouterLink` directive.
```

Comment:
```suggestion
 * with other routes. Using named outlets and secondary routes, you can target multiple outlets in
 * the same `RouterLink` directive.
```

This sentence was really confusing to me. How about this instead?